source/push-to-deploy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Push To Deploy
+order: 19
+description: Learn how to setup Push to Deploy on Cloud, so you can deploy changes to your apps just buy by performing a commit to a repository.
+---
+
+With this feature, never was easier to create an app on Galaxy. All you need to do is follow three  simple steps, then you'll have a new app deployed.
+
+<h2 id="pre-requirements">Pre requirements</h2>
+
+Today we have our integration just with GitHub. In the future we'll add more Git providers.
+
+The only requirement you need today to start using this feature is to have a repository on GitHub with a Meteor project on it. And that's it.
+
+<h2 id="deploying-a-new-app">Deploying a new app</h2>
+
+Start by going to your Cloud account [here](https://cloud.meteor.com/). Then go to *apps* > *Deploy a new app*. Once there, you'll see this page:
+
+<img src="images/push-to-deploy-first-step.png" style="width: 780px;">
+
+Go ahead and link your GitHub account to our app, just by clicking on the GitHub button.
+
+After connecting to your git provider you'll automatically go to second step. On the second step you need to chose your account from the git provider and then choose the repository where is the Meteor app that you want to deploy.
+
+<img src="images/push-to-deploy-second-step.png" style="width: 780px;">
+
+Now just click on *NEXT STEP* and you'll get on the third and last step.
+
+<img src="images/push-to-deploy-third-step.png" style="width: 780px;">
+
+On the last step you'll just need to configure the last details, so you can have your app deployed. Starting by the *Account* which defines to which Cloud account your app will be deployed to.
+
+Then you get to select which *Plan*, *Container Size*, and *Region* your app will have.
+
+On the **Deploy Options** section, you can choose which branch you'll use as your deploy branch. Every commit done on this branch will trigger a deploy. On this section you also can opt to user or not our shared [MongoDB Cluster](https://docs.meteor.com/commandline.html#meteordeploy).
```

Comment:
"Advanced options" is the label and not "deploy options".

Reply:
Advanced options is a section inside Deploy Options. I talk about the advanced options a little further.

Reply:
So it's missing to highlight it and use first letters in upper case

Reply:
:+1: